docs/guides/configuration.md

```diff
@@ -278,6 +280,45 @@ new kanban.Kanban("#root", {
 });
 ~~~
 
+#### Configuring Upload Url as Object
```

Comment:
a function. a function is an object, but it's a special kind of object. it's better to be precise and call a spade a spade, and not a gardening tool.